tests/backend/unit/test_project_structure.py

```
@@ -0,0 +1,702 @@
+                if not ast.get_docstring(tree):
+                    missing_docstrings.append(f"{module_path}: missing module docstring")
+                
+                # Check class docstrings
+                for node in ast.walk(tree):
+                    if isinstance(node, ast.ClassDef):
+                        if not ast.get_docstring(node):
+                            missing_docstrings.append(
+                                f"{module_path}:{node.name}: missing class docstring"
+                            )
+            except Exception:
+                pass
```

Ruff (0.11.9)

`try`-`except`-`pass` detected, consider logging the exception

(S110)